src/SFA.DAS.EAS.Portal.Types/SFA.DAS.EAS.Portal.Types.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why do we need a types assembly?
We have the model we use to write to Cosmos in the `SFA.DAS.EAS.Portal.Client' project , under the Models folder, and we have the client side model in `SFA.DAS.EAS.Portal.Client`, also under the Models folder.
Can we remove this assembly?

Reply:
It is the read model.  It is what should be streamed out of the database directly to the front end without any mapping.  So the UI will need to use the types as well as the event handlers.

Reply:
I understand it's the read model, but what do we gain by having a client project/assembly/nuget package and also a types project/assembly/nuget package?

It means we have an extra project in our solution (which slows down load times and increases complexity), and clients will need references to two separate packages/assemblies, when they only need a reference to one (which was the case before this types package was introduced).

So there are definite advantages to having only a single client package. What are the advantages of splitting the client out into two?

Reply:
Model moved into client project and `SFA.DAS.EAS.Portal.Types` project has been deleted. 👍